src/System.Private.Interop/src/System/Runtime/InteropServices/ComTypes/bindptr.cs

```diff
@@ -9,11 +9,16 @@ namespace System.Runtime.InteropServices.ComTypes
     [StructLayout(LayoutKind.Explicit, CharSet = CharSet.Unicode)]
     public struct BINDPTR
     {
+		[System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Security", "CA2111:PointersShouldNotBeVisible")]
         [FieldOffset(0)]
-        public IntPtr lpfuncdesc;
+        public IntPtr lpfuncdesc; // PUBLICLY DOCUMENTED lpfuncdesc field
```

Comment:
It is not obvious that this comment applies to the SuppressMessage attribute.

It is better to include this as `Justification =` in the SuppressMessage attribute.

Reply:
Also, I do not think you need to use ALL CAPS for the justification. People frequently use just:

`Justification = "Backwards compatibility"`

Reply:
Fixed. These words were copied from others places..anyway, update them according to review comment